ami_environment.yml

```diff
@@ -0,0 +1,23 @@
+name: root
+channels:
+  - pytorch
```

Comment:
Any idea if this is needed? pytorch is also available from the default channel, but not sure the difference 

(the installation instructions of pytorch seem to be to install from this channel, so maybe therefore keep it)

Reply:
can you add `cuda90` as a requirement ? otherwise it downgrades to the 8.0 version